_posts/0000-01-02-Acebuot.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
# also have to try the headers

![Promotion of Rain's artwork](https://pbs.twimg.com/media/Eym_y9DWUAEbZRE?format=jpg&name=4096x4096)

- we do them lists
- yes we d
     - we do some more
     - and we make it *italic*
     - We make it **bold**
     - __I Don't **Know** How this will work__